secrets/spi/src/main/kotlin/SecretsProvider.kt

```diff
@@ -50,8 +52,8 @@ interface SecretsProvider {
     fun removeSecret(path: Path)
 
     /**
-     * Generate a [Path] for the secret belonging to the given [organizationId], [productId] and [repositoryId], as well
-     * as the [secretName].
+     * Generate a [Path] for the secret belonging to the given [entity] with [id], as well as the [secretName].
```

Comment:
Please also describe the new default implementation.

Reply:
Done. For the record, the default implementation is not "new" in the sense that it would be doing something different than before. It's just a simpler, centralized implementation of the original logic.

I decided to move this refactoring to another preceding commit to make that more clear.